src/config/config_definition.cc

```diff
@@ -130,7 +130,7 @@
 #define DEFAULT_MARK_PLAYED_ITEMS_STRING "*"
 
 /// \brief default values for CFG_IMPORT_SYSTEM_DIRECTORIES
-static std::vector<std::string_view> excludes_fullpath {
+static std::vector<std::string> excludes_fullpath {
```

Comment:
What's the problem with `string_view`?

Reply:
Some problem in the constructor. Partial log: https://gist.github.com/neheb/73d3206c19a68ab3b2fd9d7bc35dd2d0

FWIW, this does not increase binary size. These strings are short enough to fit the short string optimization.